bot/tests/settings.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import os
+from pathlib import Path
+
+from dotenv import load_dotenv
+from pydantic import Field
+from pydantic_settings import BaseSettings
+
+logger = logging.getLogger(__name__)
+
+env_file = os.getenv('ENV_FILE')
+if env_file:
+    env_path = Path(env_file)
+else:
+    env_path = Path(__file__).parent.parent.parent / ".env"
+# pylint: disable=duplicate-code
+if env_path.exists():
+    load_dotenv(env_path)
+    logger.warning("Using dotenv file")
+else:
+    logger.info("No dotenv file found. Environment variables should be provided by the system.")
+# pylint: enable=duplicate-code
```

Comment:
dobra juz widze duplikat, spokojnie mozna to do funkcji wyjac a nie ignorowac z pylintem, dobrze ze cie opierdolil xd